README.md

```diff
@@ -17,96 +12,32 @@ Clone the source code:
-
-You can also run `npm run start:clean`, which will reset the database before starting up the Measure Repository Service server.
-
-For ease of testing, it is recommended to download [Insomnia API Client and Design Tool](https://insomnia.rest) for sending HTTP requests to the server and [Robo 3T](https://robomongo.org) as a GUI for viewing the Mongo database.
-
-When sending requests, ensure that the `"Content-type": "application/json+fhir"` header is set.
-
-### CRUD Operations
+### Overall Usage
 
-This server currently supports the following CRUD operations:
+Once you have things installed and setup in the `frontend` and `backend` directories, you can run the following in the root directory:
 
-- Read by ID to endpoint: `4_0_1/<resourceType>/<resourceId>`
-  _More functionality coming soon!_
+To start the the frontend application and backend server in parallel:
 
-### Search
-
-The Measure Repository Service server supports `Library` and `Measure` resource search by the parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#shareable-measure-repository). These parameters are:
-
-- description
-- identifier
-- name
-- status
-- title
-- url
-- version (can appear only in combination with a url search)
-
-### Package
-
-The Measure Repository Service server supports the `Library` and `Measure` `$package` operation with the `id` and `identifier` parameters and SHALL parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). Accepted parameters are:
-
-- id
-- url
-- identifier
-- version
-
-### Data Requirements
-
-The Measure Repository Service server supports the `Measure` and `Library` `$data-requirements` operations with the SHALL parameters specified in the publishable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#publishable-measure-repository). SHALL parameters are:
-
-- id
-- url
-- version
-- identifier
-
-Supported optional parameters for `Measure` are:
-
-- periodStart
-- periodEnd
-
-### Submit
-The Measure Repository Service server supports the `$submit` operation for `Measure` and `Library` resources, as specified in the Authoring Measure Repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#authoring-measure-repository). The operation does not take in any parameters. 
-
-## License
+```bash
+npm run start:all
+```
 
-Copyright 2022 The MITRE Corporation
+This starts up the Measure Repository service at `localhost:3001` and you can open the Measure Repository Service frontend application by navigating to [http://localhost:3000](http://localhost:3000) in your browser.
```

Comment:
Is port 3001 the default in `.env.example`? When I ran the `npm run start:all` command it started the Measure Repository Service on port 3000 (but that could be because that's what I have in my `.env`). We might want to set a default for the MRS port and the frontend port if we don't have them set already (ex. `next dev -p 3000` in the frontend package.json)

Reply:
Ah good point- I think I have this flip flopped. So the frontend should run on 3001 and the server should be on 3000?